### PR TITLE
Add Thai providers

### PR DESCRIPTION
**Pull request by tuwannu**

### CODE REVIEW

File: src/Faker/Provider/th_TH/Person.php [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+        '{{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}}',
+        '{{titleFemale}} {{firstNameFemale}} {{lastName}}',
+    );
+
+    /**
+     * Male first names below are extracted from soap drama characters.
+     * @see https://th.wikipedia.org/wiki/%E0%B8%A3%E0%B8%B2%E0%B8%A2%E0%B8%8A%E0%B8%B7%E0%B9%88%E0%B8%AD%E0%B8%A5%E0%B8%B0%E0%B8%84%E0%B8%A3%E0%B9%82%E0%B8%97%E0%B8%A3%E0%B8%97%E0%B8%B1%E0%B8%A8%E0%B8%99%E0%B9%8C%E0%B9%84%E0%B8%97%E0%B8%A2
```
Comment (fzaninotto) [inline src/Faker/Provider/th_TH/Person.php]: I don't see any list of names in the referenced url. What is the real datasource for the list?

File: src/Faker/Provider/th_TH/Person.php [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+     */
+    protected static $firstNameMale = array(
```
Comment (fzaninotto) [inline src/Faker/Provider/th_TH/Person.php]: Overall, this list is too short. Faker needs to generate random data and for that purpose the dictionaries must be at least 3 to 4 times larger than that.

File: src/Faker/Provider/th_TH/Person.php (hunk 1 above)
Comment (tuwannu) [inline src/Faker/Provider/th_TH/Person.php]: @fzaninotto There is no official or proper pool of Thai names available in one url. My list of names are extracted from the urls that followed the reference link. What would be your suggestion on this?

File: src/Faker/Provider/th_TH/Person.php (hunk 2 above)
Comment (tuwannu) [inline src/Faker/Provider/th_TH/Person.php]: @fzaninotto Thanks for the input. Will update accordingly.

Side question: I didn't see any guidelines on this (e.g. minimum items for a dictionary), at least not in CONTRIBUTING.md. Should this be provided somewhere? I can help develop one if needed.

Comment (fzaninotto) [inline src/Faker/Provider/th_TH/Person.php]: There is nothing in the Contributing guidelines for now. You're right: it's a good thing to add there.

**Comment by fzaninotto** [comment]
Abandoned PR, closing.

**Pull request closed without merging**